image/README.md

```diff
@@ -1,4 +1,4 @@
-[![Go Reference](https://pkg.go.dev/badge/github.com/containers/image/v5.svg)](https://pkg.go.dev/github.com/containers/image/v5) [![Build Status](https://api.cirrus-ci.com/github/containers/image.svg)](https://cirrus-ci.com/github/containers/image)
+[![Go Reference](https://pkg.go.dev/badge/go.poman.io/image/v5.svg)](https://pkg.go.dev/go.podman.io/image/v5)
```

Comment:
another typo in the badge svg link, it doesn't seem to matter for the picture but still it should be right :)